controllers/errors.controllers.js

```diff
@@ -0,0 +1,25 @@
+exports.incorrectEndpointHandler = (req, res) => {
+  res.status(404).send({ msg: "Path not found" });
+};
+
+exports.customErrorHandler = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else if (err.status === 404) {
+    res.status(404).send({ msg: "Article not found" });
```

Comment:
^^ you shouldn't need this else if, because realistically, you should be promise.rejecting with the status and msg, so therefore it should be working on the original if :)